x/gamm/simulation/sim_msgs.go

```diff
@@ -214,19 +214,26 @@ func RandomJoinSwapExternAmountIn(k keeper.Keeper, sim *simtypes.SimCtx, ctx sdk
 		return nil, fmt.Errorf("no sender with denoms %s exists", coinIn.Denom)
 	}
 
-	// cap joining pool to the pool liquidity
-	newTokenIn := osmoutils.MinCoins(sdk.NewCoins(coinIn), sdk.NewCoins(tokenIn))
+	r := sim.GetSeededRand("select random seed")
+	one_or_multiple := r.Intn(2)
+	var newTokensIn sdk.Coins
+	if one_or_multiple == 0 {
+		// cap joining pool to the pool liquidity
+		newTokensIn = osmoutils.MinCoins(sdk.NewCoins(coinIn), sdk.NewCoins(tokenIn))
+	} else {
+		newTokensIn = sdk.NewCoins(coinIn, tokenIn)
```

Comment:
Is this guaranteed to not cause errors with varying seeds? Have we tested this path specifically?